lib/puppet/pops/loader/module_loaders.rb

```diff
@@ -15,7 +15,7 @@ module Loader
 # internal layout etc.)
 #
 # A module loader is also not aware of the mapping of name to relative paths - this is performed by the
-# included module PathBasedInstantatorConfig which knows about the map from type/name to
+# included module PathBasedInstantiatorConfig which knows about the map from type/name to
```

Comment:
It looks like this never actually got implemented per https://github.com/l0b0/puppet/commit/96f3d9720b3dc0289689a090a21855ec5bc09310 @hlindberg ?

Reply:
loaders were refactored - that logic got changed along the way. Suggest updating the comment to just say: "A module loader is also not aware of the mapping of name to relative paths." and leave out any mention of how it is implemented.

Reply:
@l0b0 Can you make the suggested change?